src/backend/src/players/players.controller.ts

```diff
@@ -16,6 +16,7 @@ export class PlayersController {
     const player = await Player.create({
       userId,
       emotionId,
+      grabs: [],
```

Comment:
Does this not default to the []?

Reply:
Not sure, I couldn't find anything online, so I thought it was safer to include.

Reply:
Can you try taking it out and see if it still works?